internal/hcops/load_balancer.go

```diff
@@ -877,6 +866,21 @@ func (l *LoadBalancerOps) ReconcileHCLBServices(
 		portExists := hclbListenPorts[portNo]
 		delete(hclbListenPorts, portNo)
 
+		if port.Protocol != "" && port.Protocol != corev1.ProtocolTCP {
+            warnMsg := fmt.Sprintf(
+				"configured unsupported Hetzner Cloud load balancer protocol %s for service with name %s",
+				port.Protocol,
+				svc.Name,
+            )
+			l.Recorder.Event(
+				svc,
+				corev1.EventTypeWarning,
+				"UnsupportedProtocolConfigured",
+                warnMsg,
+			)
+			klog.Warning(warnMsg)
+		}
+
```

Comment:
Again those weird indents, is it GitHub that is messing things up or are you using space instead of tabs?

Reply:
Aah sorry, outdated changes.

Reply:
I switched to nvim recently and pretty sure I have something misconfigured there. Need to fix this. But golangci-lint fixes it, so good that we added it to pre-commit ;)